src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/_bootstrap-variables.scss

```diff
@@ -128,53 +137,53 @@ $caret-width-large:         5px !default;
 //## Customizes the `.table` component with basic values, each used across all table variations.
 
 //** Padding for `<th>`s and `<td>`s.
-$table-cell-padding:            8px !default;
+$table-cell-padding:            8px;
 //** Padding for cells in `.table-condensed`.
-$table-condensed-cell-padding:  5px !default;
+$table-condensed-cell-padding:  5px;
 
 //** Default background color used for all tables.
-$table-bg:                      transparent !default;
+$table-bg:                      transparent;
 //** Background color used for `.table-striped`.
-$table-bg-accent:               #f9f9f9 !default;
+$table-bg-accent:               #f3fcfd;
```

Comment:
Wordt inderdaad best in een variabele gestopt, maar aangezien $gray-lightest al gebruikt wordt voor #f1f7f9 wordt dat een beetje moeilijk. Foundation heeft een heleboel meer variabelen voor grijswaarden. Ter inspiratie:

![screen shot 2015-06-12 at 15 08 35](https://cloud.githubusercontent.com/assets/1300724/8130689/09648b86-1115-11e5-9841-e43a90f2b303.png)